frontend/src/views/camp/Admin.vue

```diff
@@ -56,6 +56,11 @@ export default {
   data () {
     return {}
   },
+  computed: {
+    isManager () {
+      return this.camp().role === 'manager'
```

Comment:
Ich werde durch alles durchgehen, und zumindest den String 'manager' auslagern, vielleicht finde ich auch was schöneres

Reply:
Findest du es ein Problem dass hier der String `manager` hardcoded ist? Ich habe mal angenommen dass wir uns dazu im Backend commiten, und jede Änderung daran wäre nach dem Go-Live ein breaking change.

Ich finde es mässig cool, dass dieses Feld auf dem Camp mitgeliefert wird, und so verschiedene User verschiedene Daten vom selben Camp bekommen. Das macht es in ferner Zukunft eine Spur schwieriger, gegebenenfalls einen Caching-Layer (ähnlich wie [hier](https://api-platform.com/docs/core/performance/)) einzubauen. Aber da die Berechnung via Profil, User und Collaborators recht viel komplizierter wäre habe ich mich mal dafür entschieden dieses Feld zu nutzen.